TURBO-APP-PRODUCT-GUIDE.md

```
@@ -1526,7 +1592,7 @@ Whether you're uploading files, managing domains, sharing credits, deploying web
 
 ---
 
-*Last Updated: Version 0.4.2 | October 2025*
+*Last Updated: Version 0.7.0 | October 2025*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)